cmd/minishift/cmd/addon/addon.go

```diff
@@ -28,9 +28,10 @@ const (
 )
 
 var AddonsCmd = &cobra.Command{
-	Use:   "addons SUBCOMMAND [flags]",
-	Short: "Manages Minishift add-ons",
-	Long:  "Manages Minishift add-ons. You can install, list, enable or disable Minishift add-ons.",
+	Use:     "addons SUBCOMMAND [flags]",
+	Aliases: []string{"addon"},
```

Comment:
Convenience. It gets me all the time. The plural is mainly there to align with _minikube_. Allowing both is just a nice little gimmick